app/views/foreman_virt_who_configure/api/v2/configs/main.json.rabl

```diff
@@ -6,4 +6,7 @@ attributes :name, :interval, :organization_id, :whitelist, :blacklist, :hypervis
            :hypervisor_type, :hypervisor_server, :hypervisor_username, :debug,
            :satellite_url, :proxy, :no_proxy, :status, :last_report_at, :out_of_date_at,
```

Comment:
I have a feeling the `proxy` attribute is coming back as nil on the API now - which breaks backward compatibility. What do you see when querying this API?

Reply:
it looks like it is coming back as `nil`, here is the output:

```bash
[DEBUG 2020-12-11T12:06:56 API] Response: {
                    "name" => "test",
                "interval" => 120,
         "organization_id" => 1,
               "whitelist" => "",
               "blacklist" => "",
           "hypervisor_id" => "hostname",
         "hypervisor_type" => "esx",
       "hypervisor_server" => "vcenter.toledo.satellite.lab.eng.rdu2.redhat.com",
     "hypervisor_username" => "sateng",
                   "debug" => false,
           "satellite_url" => "dhcp182-208.gsslab.rdu2.redhat.com",
                "no_proxy" => "",
                  "status" => "unknown",
          "last_report_at" => nil,
          "out_of_date_at" => nil,
     "filter_host_parents" => "",
    "exclude_host_parents" => "",
          "filtering_mode" => 0,
                      "id" => 1,
              "http_proxy" => {
          "id" => 1,
        "name" => "Food",
         "url" => "http://ginger.lab.rdu2.redhat.com"
    }
}
```

Reply:
API is now returning the URL so we don't break anything